librz/flag/flag.c

```diff
@@ -841,7 +841,12 @@ RZ_API void rz_flag_unset_all(RzFlag *f) {
 	new_spaces(f);
 }
 
-/* unset all flag items in the space with the given name */
+/**
+ * @brief Unset all flag items in the space with the given name
```

Comment:
use `\brief` `\param`, etc., like what's done in most other places. See cmd_api.c for example.

Reply:
ok cfbd43da16da887f53e113d1f19b6da51d6ecefd